index.css

```diff
@@ -1 +1,111 @@
 @import url('https://stark-ui.netlify.app/Stylesheets/main.css');
+
+.nav-brand{
+    cursor: pointer;
+    padding:0rem;
+}
+
+.nav-bar{
+    position: sticky;
+    top:0;
+    z-index: 1;
+}
+
+.poster-container{
+   position: relative;
+}
+
+.banner-button{
+    position: absolute;
+    top:-13rem;
+    left:40rem;
+    width:8rem;
+    text-align: center;
+    padding-top: 2rem;
+}
+
+.poster{
+max-width: 100%;
+margin:0 auto;
+align-content: center;
+justify-content: center;
+display: flex;
+}
```

Comment:
you can use util class  for this, as gonna be used mostly

Reply:
done